pulumi/src/config.ts

```diff
@@ -9,10 +9,10 @@ export interface Config {
   namespace: string
 }
 
-export const getConfig = async (coinstack: string): Promise<Config> => {
+export const getConfig = async (): Promise<Config> => {
   const config = (() => {
     try {
-      return new pulumi.Config('unchained').requireObject<BaseConfig>(coinstack)
+      return new pulumi.Config().requireObject<BaseConfig>("coinstack")
```

Comment:
looks like a good change, though I believe we will still need `unchained` as the project name for our `pulumi config set --path` commands to work correctly